pages.py

```diff
@@ -2,15 +2,22 @@
 from ._builtin import Page, WaitPage
 from .models import Constants
 
+
+class NotMobilePlayers(WaitPage):
+    # Only the players who weren't using their phones for this can be grouped with the rest
+    group_by_arrival_time = True
+
+
 # Description of the game: How to play and returns expected
 class Introduction(Page):
     def is_displayed(self):
         return self.round_number == 1
 
+
 class Trade(Page):
     timeout_seconds = 60
     form_model = 'player'
-    form_fields = ['trade_attempted']
+    form_fields = ['trade_attempted', 'trading']
```

Comment:
You can remove this, see my comment above.

Reply:
The 'trading' part only, not the whole line.

Reply:
Kristian asked for at least one field that detects bot users (not the ones created for the game)